src/fusion_power_plant_tests.cc

```diff
@@ -37,18 +64,442 @@ TEST_F(FusionPowerPlantTest, Print) {
+      " <blanket_turnover_fraction>0.03</blanket_turnover_fraction>"
+      " <he3_outcommod>Helium_3</he3_outcommod>";
+
+  int simdur = 3;
+  cyclus::MockSim sim(cyclus::AgentSpec(":tricycle:FusionPowerPlant"), config, simdur);
+
+  sim.AddRecipe("tritium", tritium());
+  sim.AddRecipe("enriched_lithium", enriched_lithium());
+
+  sim.AddSource("Enriched_Lithium").recipe("enriched_lithium").Finalize();
+
+  int id = sim.Run();
+
+  // Under these conditions, we expect the reactor to never start, meaning it
+  // never sequesteres tritium.
+  std::vector<Cond> conds;
+  conds.push_back(Cond("Time", "==", std::string("2")));
+  QueryResult qr = sim.db().Query("FPPInventories", &conds);
+  double seq_trit = qr.GetVal<double>("TritiumSequestered");
+
+  EXPECT_EQ(0, seq_trit);
+}
+
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+TEST_F(FusionPowerPlantTest, DecayInventoryExtractHelium) {
+  // Test behaviors of the DecayInventory and ExtractHelium function here
+  EXPECT_NO_THROW(facility->DecayInventories());
+
+  // We use unintuitive values for reserve inventory and startup inventory here
+  // because they were the values we were originally testing with, and we had
+  // already done all the calculations.
+  std::string config =
+      " <fusion_power>300</fusion_power> "
+      " <TBR>1.00</TBR> "
+      " <reserve_inventory>6.0</reserve_inventory>"
+      " <sequestered_equilibrium>2.121</sequestered_equilibrium>"
+      " <fuel_incommod>Tritium</fuel_incommod>"
+      " <blanket_incommod>Enriched_Lithium</blanket_incommod>"
+      " <blanket_outcommod>Depleted_Lithium</blanket_outcommod>"
+      " <blanket_inrecipe>enriched_lithium</blanket_inrecipe>"
+      " <blanket_size>1000</blanket_size>"
+      " <he3_outcommod>Helium_3</he3_outcommod>";
+
+
+  int simdur = 2;
+  cyclus::MockSim sim(cyclus::AgentSpec(":tricycle:FusionPowerPlant"), config, simdur);
+
+  sim.AddRecipe("tritium", tritium());
+  sim.AddRecipe("enriched_lithium", enriched_lithium());
+
+  sim.AddSource("Tritium").recipe("tritium").Finalize();
+  sim.AddSource("Enriched_Lithium").recipe("enriched_lithium").Finalize();
+
+  int id = sim.Run();
+
+  std::vector<Cond> conds;
+  conds.push_back(Cond("Time", "==", std::string("1")));
+  QueryResult qr = sim.db().Query("FPPInventories", &conds);
+  double he3 = qr.GetVal<double>("HeliumExcess");
+
+  double reserve = 6.0;
+  double sequestered = 2.121;
+
+  double init_quant = reserve + sequestered;
+  // Lambda in base 2, not base e (see Decay.cc for more info)
+  double lambda = 2.57208504984001213e-09; 
+  double t = 2629846;
+
+  double expected_decay = (init_quant - sequestered) 
+              - (init_quant - sequestered) * std::pow(2, -lambda * t);
+
+  EXPECT_NEAR(expected_decay, he3, 1e-6);
+}
+
+// - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+TEST_F(FusionPowerPlantTest, Li7EdgeCases) {
+  // Test that FPP does the same thing regardless of Li-7 Contribution
```

Comment:
Should something be different this this case?  If there is no difference ever, why include it at all?

Reply:
The point of this test was more to ensure that if users set different Li7 contribution the amount of tritium they get out doesn't change, which it shouldn't. Thinking about our design decision, I don't fully remember why we added the ability to change how much the Li-7 contributes to the breeding reaction, but I think it had to do with some nuance of blanket design? I remember Matthew running a whole openMC calculation which we based the default on, but beyond that it was so long ago that I forget.

Reply:
Ultimately, the blanket waste will have different compositions based on this, I guess.